__tests__/data/am_output.json

```diff
@@ -1,4 +1,60 @@
 [
+  {
+    "version": "1",
+    "type": "REST",
+    "method": "POST",
+    "endpoint": "https://api.amplitude.com/2/httpapi",
+    "headers": {
+      "Content-Type": "application/json"
+    },
+    "params": {},
+    "body": {
+      "JSON": {
+        "api_key": "abcde",
+        "events": [
+          {
+            "os_name": "Chrome",
+            "os_version": "test os",
+            "device_model": "Mac",
+            "platform": "Web",
+            "device_id": "lynnanderson@smith.net",
+            "app_name": "RudderLabs JavaScript SDK",
+            "app_version": "1.0.0",
+            "language": "en-US",
+            "session_id": "3049dc4c-5a95-4ccd-a3e7-d74a7e411f22",
+            "insert_id": "84e26acc-56a5-4835-8233-591137fca468",
+            "ip": "0.0.0.0",
+            "user_properties": {
```

Comment:
This we did not add in integration-info for mapping. For transformer, its fine I think. But can you confirm as VDM in reverse ETL one cannot sync `user_properties` without say using user transformation?
Ref other PR: https://github.com/rudderlabs/rudder-integrations-info/pull/52/files#r808463508

Reply:
The same explanation as I had mentioned in the comment of https://github.com/rudderlabs/rudder-integrations-info/pull/52/files#r808463508

Reply:
Discussed. Resolving